.travis.yml

```diff
@@ -11,6 +11,8 @@ matrix:
         - os: linux
           python: 3.5
         - os: linux
+          python: 3.6
+        - os: linux
```

Comment:
Any idea if 3.6 is available for the OS X builds? Since they upload wheels it would probably be good to add it if possible.